MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -12,21 +12,28 @@ internal class MiniAppWebView: WKWebView {
         return config
     }
 
-    convenience init(miniAppId: String, versionId: String) {
+    convenience init(miniAppId: String, versionId: String, queryParams: String? = nil) {
         let schemeName = Constants.miniAppSchemePrefix + miniAppId
-        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName)!)
+        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName + MiniAppWebView.getQueryParams(queryParams: queryParams))!)
         let config = MiniAppWebView.defaultConfig()
         config.setURLSchemeHandler(URLSchemeHandler(versionId: versionId), forURLScheme: schemeName)
         self.init(frame: .zero, configuration: config)
         commonInit(urlRequest: urlRequest)
     }
 
-    convenience init(miniAppURL: URL) {
+    convenience init(miniAppURL: URL, queryParams: String? = nil) {
         let urlRequest = URLRequest(url: miniAppURL)
         self.init(frame: .zero, configuration: MiniAppWebView.defaultConfig())
         commonInit(urlRequest: urlRequest)
     }
 
+    private static func getQueryParams(queryParams: String?) -> String {
+        guard let param = queryParams, let queryString =  param.encodeURLParam() else {
+            return ""
+        }
+        return "?" + queryString
```

Comment:
On Android, I think that @khairul-alam-licon made it so the `?` mark is added only if the provided string doesn't already start with a `?`. This is because it's very unlikely that a mini app would purposely want an extra `?` at the start of the query.

Could you do something similar on iOS so that the behavior is matching across platforms?

Reply:
😂😂😂
I did make those changes already in this PR, Check here https://github.com/rakutentech/ios-miniapp/pull/199/commits/e889acaf99a7b57d8870c003e6237926fc17a7e7#diff-b98abf2a7523c021d82b52ed01322d682e337911e1dd79400645ef0001694185R30

But @khairul-alam-licon said they are not doing like that in Android so I had remove it.

Reply:
Oh, sorry I didn't realize it was removed on Android.

That's fine with me as long as it matches across platforms.